packages/daimo-api/src/server/tokenRegistry.ts

```diff
@@ -28,19 +29,35 @@ interface CoinGeckoToken {
 
 const customOverrides = [baseUSDC, baseUSDbC] as ForeignToken[];
 
-// Token Registry sorted by chain id.
+/**
+ * Token Registry sorted by chain id.
+ */
 export class TokenRegistry {
+  /** Map of chainId to token address to foreign token type. */
   private foreignTokensByChain = new Map<number, Map<Address, ForeignToken>>();
 
   private defaultChainId = chainConfig.chainL2.id;
 
   // No token registry on testnet
+  // TODO: add all CCTP chains
   private chains = chainConfig.chainL2.testnet ? [] : [this.defaultChainId];
 
   public async load() {
+    // First, load all Daimo common tokens (CCTP and native tokens) into reg
+    for (const [chainId, tokens] of chainToForeignTokens) {
+      const foreignTokens = new Map<Address, ForeignToken>();
+      for (const token of tokens) {
+        foreignTokens.set(token.token, token);
+      }
+      this.foreignTokensByChain.set(chainId, foreignTokens);
+    }
+
+    // Then, fetch from CoinGecko
```

Comment:
don't have to anymore--we have the full coingecko list across all supported chains, exported by `@daimo/contract`